index.md

```diff
@@ -3,7 +3,23 @@ layout: default
 title: BMLT Days 2019
 ---
 
-join info: [https://zoom.us/j/613507415](https://zoom.us/j/613507415)
+*Join info*: [https://zoom.us/j/613507415](https://zoom.us/j/613507415)
+
+*One tap mobile*
+
+[+16465588656,,613507415#](tel:+16465588656,,613507415#) US (New York)
```

Comment:
I don't think you can render links with `tel:` in markdown.  At least it doesn't seem to be working.

Reply:
it worked here, https://pjaudiomv.github.io/jft/ in my viewer in intellij the link didnt work but it did on github pages

Reply:
It might be just the preview mode on gh.

Reply:
Looks good on deploy, thanks!